pandas/_libs/groupby.pyx

```diff
@@ -1270,11 +1274,14 @@ cdef group_cummin_max(groupby_t[:, ::1] out,
     cdef:
         Py_ssize_t i, j, N, K, size
         groupby_t val, mval
-        ndarray[groupby_t, ndim=2] accum
+        groupby_t[:, ::1] accum
         intp_t lab
+        bint val_is_nan, use_mask
+
+    use_mask = mask is not None
 
     N, K = (<object>values).shape
-    accum = np.empty((ngroups, K), dtype=np.asarray(values).dtype)
+    accum = np.empty((ngroups, K), dtype=np.asarray(values).dtype, order='C')
```

Comment:
why `np.asarray(values)` it is *already* an ndarray, so just `.dtype`

Reply:
thanks, changed